docs/_docs/interface.md

```diff
@@ -8,29 +8,28 @@ toc: false
 
 ## Home
 
-When you browse to the root of your registry, it's a fairly expected interface with
-links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center"
+When you browse to your registry's root, it's a reasonably expected interface with
+links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center".
 
 ![teams.png](../assets/img/interface/home.png)
 
-When you sign in with social authentication (e.g., GitHub) you are asked to sign standard terms of service.
-This basically says that the software is provided for you, free to use, and you are responsible
+When you sign in with social authentication (e.g., GitHub), you first must sign the standard terms of service (TOS).
+The TOS asserts that the software is provided for you, free to use, and you are responsible
 for taking care of your registry and are accountable for the containers within it.
 
 ![teams.png](../assets/img/interface/terms.png)
 
-When you agree, the robots welcome you!
+When you agree, robots welcome you!
 
 ![teams.png](../assets/img/interface/terms-welcome.png)
 
-
 ## Collections 
 
 A collection is a set of containers under the same namespace. For example, `dinosaur/avocado` and `dinosaur/banana` could be two containers in the `dinosaur` collection. You can browse all collections in the main collection view:
 
 ![collections.png](../assets/img/collections.png)
 
-and browse the containers within a collection by clicking on it. Here is a collection with one container:
+And browse the containers within a collection by clicking on it. Here is a collection with one container:
```

Comment:
This was a continuation of the previous sentence (hence lowercase)

Reply:
Fixed